src/translations/en.json

```diff
@@ -576,7 +576,7 @@
     "loadingReport": "Loading logs in progress...",
 
     "ComponentLibrary": "Component library selection",
-    "Advanced": "Advanced",
+    "UserSettings": "User settings",
```

Comment:
I know there is one translation for the menu item, and one for the Dialog Title but it's exactly the same translation, same text. Maybe use only one it's ok

Reply:
I do not find the other translation. What are you refering to ?

Reply:
Ok got it